### DISCUSSION TITLE
.\Winget.exe does nothing when is run

### DISCUSSION THREAD

**Discussion by Swe-it** [microsoft/winget-cli#3146] [General]
Anyone else that have this problem when running Winget as System? Nothing happens.

`PS C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.19.10173.0_x64__8wekyb3d8bbwe> dir

-a----        2023-03-31     14:32        4534272 WindowsPackageManager.dll                                                                                                          
-a----        2023-03-31     14:32          46592 WindowsPackageManagerServer.exe                                                                                                    
-a----        2023-03-31     14:32          12288 winget.exe                                                                                                                         

PS C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.19.10173.0_x64__8wekyb3d8bbwe> .\winget.exe search 7zip

PS C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_1.19.10173.0_x64__8wekyb3d8bbwe> `

**Comment by Trenly** [comment]
Winget currently does not support being run in the system context.

* https://github.com/microsoft/winget-cli/discussions/962

**Reply by Swe-it** [reply]
It does work with System but sometimes I get the above result, usually with new deployed computers.
